src/clients/ntp-client_auto.rb

```diff
@@ -63,6 +68,19 @@ def main
         @ret = NtpClient.Import(@param)
       # Return actual state
       elsif @func == "Export"
+        # The ntp.conf has to be read after the
+        # package has been installed in the installation
+        # mode (bnc#928987)
+        # Mode.config : true during the installation when
+        #               cloning the just installed system
+        if Mode.config && Stage.initial
+          ntp_conf = "/etc/ntp.conf"
+          # copy ntp.conf from the installed system to
+          # running system
+          `/usr/bin/cp #{File.join(Installation.destdir, ntp_conf)} #{ntp_conf}`
```

Comment:
I'd prefer [FileUtils::cp](http://ruby-doc.org/stdlib-2.1.0/libdoc/fileutils/rdoc/FileUtils.html#method-c-cp) here, you already added `require "fileutils"` few lines above... :wink: